src/lib/styles/tailwindColors.ts

```diff
@@ -1,28 +1,25 @@
 // Used for storybook.
 
 export default {
```

Comment:
Separate issue: I think we removed this from the package exports, but I think we still want it.

Reply:
@wilwade Before, we were importing this into the tailwind config and using it to set the theme colors. Now, we're doing that in the css file and this file is being used to easily loop through the colors to display them in the storybook. Because of that, I believe we don't need it in the package.

Reply:
I was thinking it could be useful for setting colors and such in js if needed, but I guess even those should be done via the css variables.